components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/provisioning/impl/DefaultProvisioningHandler.java

```diff
@@ -149,6 +155,26 @@ public void handle(List<String> roles, String subject, Map<String, String> attri
                  */
                 IdentityUtil.threadLocalProperties.get().put(FrameworkConstants.JIT_PROVISIONING_FLOW, true);
                 if (!userClaims.isEmpty()) {
+                    /*
+                    In the syncing process of existing claim mappings with IDP claim mappings for JIT provisioned user,
+                    To delete corresponding existing claim mapping, if any IDP claim mapping is absence.
+                     */
+                    Claim[] existingUserClaimList = userStoreManager.getUserClaimValues(
+                            UserCoreUtil.removeDomainFromName(username), null);
```

Comment:
let's pass "UserCoreConstants.DEFAULT_PROFILE" instead of null.

Reply:
Addressed with 2a770299d41a205516625900b8cb0c64e415b6e9